config/software/chef.rb

```diff
@@ -64,11 +64,7 @@
   excluded_groups << "ruby_shadow" if aix?
 
   # install the whole bundle first
-  bundle "install --without #{excluded_groups.join(' ')}", env: env
-
-  # Install components that live inside Chef's git repo. For now this is just
-  # 'chef-config'
-  bundle "exec rake install_components", env: env
+  bundle install
```

Comment:
actually that should be `bundle "install", env: env`

Reply:
yep